lib/msal-react/FAQ.md

```diff
@@ -114,7 +114,7 @@ function Example() {
     const { result, error, login } = useMsalAuthentication(InteractionType.Popup);
 
     useEffect(() => {
-        if (error && error.errorMessage.indexOf("AADB2C90118") > -1) {
+        if (error && error.message.indexOf("AADB2C90118") > -1) {
```

Comment:
I don't think this is correct. This error should be of type `ServerError` which has [this shape](https://azuread.github.io/microsoft-authentication-library-for-js/ref/classes/_azure_msal_common.servererror.html). Are you having problems with this?

Reply:
Hi @tnorling, yes you are right the error object returned by useMsalAuthentication does have the correct shape `ServerError`. Will update my PR.

Reply:
I have updated this PR @tnorling